### DISCUSSION TITLE
Absolute paths for shared files/directories

### DISCUSSION THREAD

**Discussion by pavelgross** [deployphp/deployer#3069] [Help needed]
Hi,
I use deployer v7

Is it possible to use absolute symlinks instead of releative for the shared files & dirs? The data should be shared via nfs and the server setup has problems with the relative paths.
If there is no parameter for this, would customization via after hook for deploy:shared be good way to change the symlinks?

Thx

**Comment by antonmedv** [comment]
Set https://deployer.org/docs/7.x/recipe/common#use_relative_symlink to fasle.

**Reply by pavelgross** [reply]
Cool, thanks. I didn't see that it could go so easily.
